packages/appeals-service-api/src/main.js

```diff
@@ -10,7 +10,8 @@ const mongodb = require('./db/db');
 
 async function main() {
 	try {
-		appInsights.setup().start();
+		appInsights.setup().setAutoDependencyCorrelation(true).setSendLiveMetrics(true).start();
+		appInsights.defaultClient.setAutoPopulateAzureProperties(true);
```

Comment:
I think the `setAutoPopulateAzureProperties()` call needs to be made between the setup steps and the call to `start()` however, from looking into the configuration a bit more, it looks like auto-populating the app name from the Azure properties may not work on container apps so I think this will need to be rewritten as:
```
appInsights.setup().setAutoDependencyCorrelation(true).setSendLiveMetrics(true);
appInsights.defaultClient.context.tags[appInsights.defaultClient.context.keys.cloudRole] = "appeals-service-api";
appInsights.start();
```